python/xorbits/_mars/dataframe/utils.py

```diff
@@ -104,6 +104,28 @@ def _get_index_for_on(
         return [idx_to_grouped.get(i, pd.Index([])) for i in range(size)]
 
 
+def _get_index_map(
```

Comment:
Better have this method covered by tests.

Reply:
UT added.